index.html

```diff
@@ -392,7 +394,7 @@
   ctx.font = radius*0.20 + "px arial";
   ctx.textBaseline="middle";
   ctx.textAlign="center";
-  for(num = (max % incr === 0) ? max - incr : max - max % incr; num >= 0; num -= incr){

```

Comment:
I tried to look up how this works in github, since the workflow is a bit different than the one we use at work, but I'm still not very sure. It may be that once the review is approved, it will require updating with master's changes before merging? But it seems silly for you not to be able to do that before the review process.